renovate.json

```
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended"
+  ]
```

By default renovate will completely ignore test projects. We also want dependency updates for our test projects so the following override neets to be added.

```suggestion
  ],
  "ignorePresets": [
    ":ignoreModulesAndTests"
  ]
```